src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -9,61 +9,109 @@ import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 import { assetActions } from '../../data/constants/actionTypes';
-import { clearAssetsStatus, deleteAsset } from '../../data/actions/assets';
+import { clearAssetsStatus, deleteAsset, sortUpdate } from '../../data/actions/assets';
 
 export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayAssetsList: [],
       modalOpen: false,
       statusAlertOpen: false,
       assetToDelete: {},
+      deletedAsset: {},
```

Comment:
deletedAsset stores a copy of assetToDelete after the asset has been deleted from the Redux store. We need this information to populate the status alert with data about the already deleted asset but didn't want to rely on a reference to a now-deleted asset. It also allows us to clear the value of assetToDelete after the delete, keeping the semantics of their names correct.